docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -489,128 +724,209 @@ Combinations of environment variables and their uses.
+After starting the yugabyted processes on all nodes, configure the data placement constraint of the cluster as follows:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=region
+./bin/yugabyted configure data_placement --fault_tolerance=region
 ```
 
-This command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, the command configures the cluster to survive at least 1 region failure. Otherwise, it outputs a warning message.
+The preceding command automatically determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are three or more regions available in the cluster, the `configure` command configures the cluster to survive at least one availability region failure. Otherwise, it outputs a warning message.
 
 The replication factor of the cluster defaults to 3.
 
-You can set the data placement constraint manually using the `--data_placement_constraint` flag, which takes the comma-separated value of `cloud.region.zone` as follows:
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone`. For example:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+```
+
+You can set the replication factor of the cluster manually using the `--rf` flag. For example:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+```
+
+### Enabling and Disabling encryption at rest
+
+Create a cluster using the desired deployment example.
+
+To enable encryption at rest in a deployed local cluster, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --enable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
 
+To enable encryption at rest in a deployed multi-zone or multi-region cluster, run the following command from any VM:
 ```sh
-./bin/yugabyted configure --fault_tolerance=region --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+./bin/yugabyted configure encrypt_at_rest --enable
 ```
 
-You can set the replication factor of the cluster manually using the `--rf` flag as follows:
+To disable encryption at rest in a local cluster with encryption at rest enabled, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --disable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
 
+To disbale encryption-at-rest in a multi-zone or multi-region cluster with encryption-at-rest enabled, run the following command from any VM:
```

Comment:
To disable encryption at rest in a multi-zone or multi-region cluster with this type of encryption enabled, run the following command from any VM: